build.sbt

```diff
@@ -24,14 +24,16 @@ lazy val freestyleJVM = freestyle.jvm
 lazy val freestyleJS  = freestyle.js
 
 lazy val tagless = (crossProject in file("freestyle-tagless"))
-  .dependsOn(freestyle)
+  .dependsOn(freestyle % "test -> compile")
   .settings(name := "freestyle-tagless")
   .jsSettings(sharedJsSettings: _*)
   .crossDepSettings(commonDeps: _*)
   .settings(resolvers += Resolver.bintrayRepo("kailuowang", "maven"))
-  .settings(
-    libraryDependencies += "com.kailuowang" %%% "mainecoon-core" % "0.0.5"
-  )
+  .settings( libraryDependencies ++= commonDeps ++ Seq(
+    "com.kailuowang" %%% "mainecoon-core" % "0.0.5",
+    %%%("cats-core"),
+    %%%("cats-free")
```

Comment:
I think these need ot be added in `crosDepSettings` ?

Also, if you include `cats-free` you can omit `cats-core`.